stellarphot/photometry/source_detection.py

```diff
@@ -70,29 +74,29 @@ def compute_fwhm(ccd, sources, fwhm_estimate=5,
     sources : `astropy.table.Table`
         An astropy table of the positions of sources in the image.
 
-    fwhm_estimate : float, optional
+    fwhm_estimate : float, optional (default=5)
```

Comment:
I think we don't need the default here explicitly since it is in the function signature...